### PR TITLE
Fix unit test coverage script

### PR DESCRIPTION
**Pull request by antonrohr**
The script did not work anymore since the folder structure change to `/src`. This PR updates this script

### CODE REVIEW

File: contrib/unit_test_coverage/generate_coverage_report.sh [hunk 1]
```diff
@@ -81,10 +83,11 @@ SUM_LINES_COVERED=0
 SUM_REGIONS_COUNT=0
 SUM_REGIONS_COVERED=0
 
-for BUILD_SUB_DIR in $(ls -d $BUILD_DIR/*/ | grep -v "$BUILD_SUB_DIR_FILTER")
+for COMPONENT_DIR in $(ls -d $BUILD_DIR/src/*/ | grep -v "$BUILD_SRC_DIR_FILTER")
 do
-  COMPONENT=$(basename $BUILD_SUB_DIR)
-  if ls $BUILD_DIR/$COMPONENT/*.profraw 1> /dev/null 2>&1; then
+  echo $COMPONENT_DIR
```
Comment (beckerhe) [inline contrib/unit_test_coverage/generate_coverage_report.sh:88]: Looks like a debug message. Intended?

File: contrib/unit_test_coverage/generate_coverage_report.sh [hunk 2]
```diff
@@ -104,7 +107,7 @@ do
     -output-dir=$OUTPUT_DIR/coverage_$COMPONENT \
     -instr-profile=$BUILD_DIR/$COMPONENT.profdata \
     $FORMATTED_BINARY_FILES \
-    $SOURCE_FILES >/dev/null 2>&1
+    $SOURCE_FILES
```
Comment (beckerhe) [inline contrib/unit_test_coverage/generate_coverage_report.sh:110]: Intended?

**Review by beckerhe — APPROVED** [review]
LGTM

File: contrib/unit_test_coverage/generate_coverage_report.sh (hunk 2 above)
Comment (antonrohr) [reply contrib/unit_test_coverage/generate_coverage_report.sh:110]: Yes. It produces warnings and I decided to display them. Might help debugging in the future

File: contrib/unit_test_coverage/generate_coverage_report.sh (hunk 1 above)
Comment (antonrohr) [reply contrib/unit_test_coverage/generate_coverage_report.sh:88]: Good catch! Thank you

**Pull request merged**